packages/launchpad/cypress/e2e/maxExecution-launchpad.cy.ts

```diff
@@ -0,0 +1,24 @@
+describe('maxExecution: launchpad', () => {
+  beforeEach(() => {
+    cy.scaffoldProject('todos')
+    cy.openProject('todos')
+  })
+
+  it('loads through to the browser screen when the network is slow', () => {
+    cy.setFetchDelay(10000)
```

Comment:
dang, i  wish we could use `cy.clock()` here. hello slow test!

Reply:
Haha, just fixed this locally, keeping track of the timers & added a method to clear them `cy.clearFetchDelay`